pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -545,6 +545,9 @@ func (p *criStatsProvider) getContainerUsageNanoCores(stats *runtimeapi.Containe
 	}
 
 	nanoSeconds := stats.Cpu.Timestamp - cached.Timestamp
```

Comment:
can we log the timestamps when the nanoSeconds is 0 so that we can see what their actual values are?

Reply:
I meant log both stats.Cpu.Timestamp and cached.Timestamp